WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -217,7 +217,7 @@ private async Task OnDoTransactionBroadcastAsync()
 				MainWindowViewModel.Instance.StatusBar.TryAddStatus(StatusType.BroadcastingTransaction);
 				await Task.Run(async () => await Global.TransactionBroadcaster.SendTransactionAsync(transaction));
 
-				NotificationHelpers.Success("Transaction is successfully broadcasted!", "");
+				NotificationHelpers.Success("Transaction was broadcasted.");
```

Comment:
```suggestion
				NotificationHelpers.Success("Transaction was broadcast.");
```

https://en.wiktionary.org/wiki/broadcasted
https://english.stackexchange.com/questions/33207/broadcast-or-broadcasted

Reply:
Interesting but for not native speakers it seems wrong, like me. For native speakers broadcasted are acceptable. So I am on not touching it.

Reply:
I'm dead. I have not been aware of it either, especially have a hard time to believe that the latter is more used. But no matter how I look at the data, broadcast is indeed the preferred word :/